File: models/Sentence.py

class Sentence:

    # предложение. базовый элемент истории.

    def __init__(self, subject, action, object, id):
        self._subject = subject      # субъект события
        self._action = action        # действие
        self._object = object        # объект события
        self._id = id            # идентификатор предложения

    def tell(self):
        return self._subject +' '+self._action +' '+self._object+'.'
    
    def id(self):
        return self._id

File: BrocaZone/models/Sentence.cs
using System;

namespace BrocaZone.models;

public class Sentence
{
    public Sentence(Guid subjectId, Action action, Guid objectId, Guid id)
    {
        Id = id;
        SubjectId = subjectId;
        Action = action;
        ObjectId = objectId;
    }

    public Guid Id { get; set; }
    public Guid SubjectId { get; set; }
    public Action Action { get; set; }
    public Guid ObjectId{ get; set; }

}
